lib/library.go

```diff
@@ -210,41 +212,44 @@ func Logout() *C.char {
 	return makeJSONResponse(err)
 }
 
-//ApproveSignRequestWithArgs instructs backend to complete sending of a given transaction.
-// gas and gasPrice will be overrided with the given values before signing the
-// transaction.
-//export ApproveSignRequestWithArgs
-func ApproveSignRequestWithArgs(id, password *C.char, gas, gasPrice C.longlong) *C.char {
-	result := statusBackend.ApproveSignRequestWithArgs(C.GoString(id), C.GoString(password), int64(gas), int64(gasPrice))
-
-	return prepareApproveSignRequestResponse(result, id)
+// SendTransaction converts RPC args and calls backend.SendTransaction
+func SendTransaction(txArgsJSON, password *C.char) *C.char {
+	txArgs, err := transactions.UnmarshalSendTxRPCParams(C.GoString(txArgsJSON))
```

Comment:
@andrey I made similar interfaces for SendTransaction and SignMessage here. Basically unmarshalling JSON into the data structures that we use. Let me know if these will be workable on the status-react side or if we should modify them to be more like https://github.com/status-im/status-go/blob/develop/transactions/types.go#L77